actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1496,6 +1496,12 @@ def time_field(object_name, method, options = {})
       #   datetime_field("user", "born_on", min: "2014-05-20T00:00:00")
       #   # => <input id="user_born_on" name="user[born_on]" type="datetime-local" min="2014-05-20T00:00:00.000" />
       #
+      # By default, provided datetimes will be formatted including seconds. You can render just the date, hour
+      # and minute by passing `include_seconds: false`.
```

Comment:
Need to use `<tt>` here because RDoc doesn't support backticks:

```suggestion
      # By default, provided datetimes will be formatted including seconds. You can render just the date, hour,
      # and minute by passing <tt>include_seconds: false</tt>.
```

(Also, snuck in an [Oxford comma](https://guides.rubyonrails.org/api_documentation_guidelines.html#oxford-comma).)